### PR TITLE
Add fields to the account entity

### LINKED ISSUE #24: Add fields to the Account entity
We need the following fields for the Account table:
- expiration date (month and year). Research common approaches.
- CVV (3 digits)
- currency ('USD', 'EUR'). Use Enum for values.

1. update the table using Liquibase (don't remove the existing table).
2. Update the entity.

### LINKED ISSUE #25: Add Account records into Database
We need to have some predefined values for users' accounts.
Add the following account using Liquibase changelog:
1. User 1: 2 accounts, USD and EUR, 1000 and 500 balances, expiration date 05/24 and 03/22
2. User 2: 2 accounts, USD and USD, 150 and 0 balances, expiration date 02/24 and 05/21

### PR DESCRIPTION
**Pull request by andy-sam**
Fixes #24 
Fixes #25

### CODE REVIEW

**Comment by andy-sam** [comment]
I made 1 issue from 2 yet

File: src/main/java/com/codesarge/banking/entities/Account.java [hunk 1]
```diff
@@ -24,61 +36,27 @@
...
+    @Column (nullable = false)
+    @Enumerated(EnumType.STRING)
+    private Currency currency;
+
+    @Column(nullable = false)
+    private String cvv;
+
+    @Column (name = "exp_date")
+    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="dd-MM-yyyy")
+    private Date date;
```
Comment (alex-anakin) [inline src/main/java/com/codesarge/banking/entities/Account.java:48]: name the field `expirationDate`

File: src/main/java/com/codesarge/banking/entities/AccountDto.java [hunk 2]
```diff
@@ -12,11 +15,17 @@
     private Long balance;
     private Long id;
     private Long userId;
+    private Currency currency;
+    private String cvv;
+    private Date date;
```
Comment (alex-anakin) [inline src/main/java/com/codesarge/banking/entities/AccountDto.java:20]: expirationDate

File: src/main/resources/db/changelog/changelog-2021-07-04-01.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+                    nullable: false
+              - column:
+                  name: cvv
+                  type: varchar(3)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: exp_date
+                  type: date
+#                  defaultValue: now()
```
Comment (alex-anakin) [inline src/main/resources/db/changelog/changelog-2021-07-04-01.yaml:21]: remove

File: src/main/java/com/codesarge/banking/entities/Account.java [hunk 4]
```diff
@@ -24,61 +36,27 @@
...
+
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "user.id")
     private User user;
 
     @Autowired
-    public Account(User user) {
-        this.number = openNewAccount();
+    public Account(User user, Currency currency) {
```
Comment (alex-anakin) [inline src/main/java/com/codesarge/banking/entities/Account.java:55]: add expDate initialization using Java Date API:
now() + 2 years

File: src/main/resources/db/changelog/changelog-2021-07-05-01.yaml [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                  value: 1
+              - column:
+                  name: currency
+                  value: "USD"
+              - column:
+                  name: cvv
+                  value: "000"
+              - column:
+                  name: exp_date
+                  value: now()
```
Comment (alex-anakin) [inline src/main/resources/db/changelog/changelog-2021-07-05-01.yaml:26]: use dates from the task, for example, '2023-07-00'

**Review by alex-anakin — CHANGES_REQUESTED** [review]

File: src/main/java/com/codesarge/banking/entities/Account.java (hunk 1 above)
Comment (andy-sam) [reply src/main/java/com/codesarge/banking/entities/Account.java:48]: done

File: src/main/java/com/codesarge/banking/entities/AccountDto.java (hunk 2 above)
Comment (andy-sam) [reply src/main/java/com/codesarge/banking/entities/AccountDto.java:20]: done

File: src/main/resources/db/changelog/changelog-2021-07-04-01.yaml (hunk 3 above)
Comment (andy-sam) [reply src/main/resources/db/changelog/changelog-2021-07-04-01.yaml:21]: done

File: src/main/java/com/codesarge/banking/entities/Account.java (hunk 4 above)
Comment (andy-sam) [reply src/main/java/com/codesarge/banking/entities/Account.java:55]: done

File: src/main/resources/db/changelog/changelog-2021-07-05-01.yaml (hunk 5 above)
Comment (andy-sam) [reply src/main/resources/db/changelog/changelog-2021-07-05-01.yaml:26]: done

**Review by alex-anakin — APPROVED** [review]

**Pull request merged**